contracts/governance/src/governance.rs

```diff
@@ -282,4 +279,53 @@ impl GovernanceContract {
 
         Ok(())
     }
+    /// Update the authority public address of the Governance Contract
+    pub fn update_authority(&mut self, key: &[u8; 96]) -> Result<(), Error> {
+        self.authority = Self::bls_public_key(key)?;
+
+        Ok(())
+    }
+    /// Update the broker public address of the Governance Contract
+    pub fn update_broker(&mut self, key: &[u8; 96]) -> Result<(), Error> {
+        self.broker = Self::bls_public_key(key)?;
+
+        Ok(())
+    }
+}
+
+// write unit test for this module
+#[cfg(test)]
+mod tests {
+    use dusk_bls12_381::G2Affine;
+
+    use super::*;
+
+    #[test]
+    fn check_update_authority() {
+        let mut contract = GovernanceContract::default();
+
+        assert_eq!(contract.authority, BlsPublicKey::default());
+        let key = G2Affine::generator().to_bytes();
+
+        contract.update_authority(&key).unwrap();
+
+        assert_eq!(
+            contract.authority,
+            GovernanceContract::bls_public_key(&key).unwrap()
+        );
```

Comment:
I would have this enriched with the signature verification against the updated authority

Reply:
This is gonna be handled later. I will create a issue and let this PR merge for now 

https://github.com/dusk-network/rusk/issues/814